o-fish-ios/Views/Components/Popover/PopoverManager.swift

```diff
@@ -80,3 +80,9 @@ class PopoverManager {
 //        )
 //    }
 //}
+
+struct PopoverManager_Previews: PreviewProvider {
+    static var previews: some View {
```

Comment:
It's incorrect preview

Reply:
That was added automatically, so I don't think it's even needed